dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKRunner.java

```diff
@@ -258,7 +258,8 @@ private void runTCK() throws Exception {
         List<String> dependencyOutput = runDependencyCmd();
         TCKJarInfo tckJarInfo = TCKUtilities.getTCKJarInfo(this.type, dependencyOutput);
         TCKResultsInfo resultsInfo = new TCKResultsInfo(this.type, this.specName, this.server, tckJarInfo);
-        TCKResultsWriter.preparePublicationFile(resultsInfo);
+        String repeat = RepeatTestFilter.getRepeatActionsAsString();
```

Comment:
in theory, it is possible to have multiple repeat actions active at once. Not sure what would happen then.